Source/Core/DolphinQt/CMakeLists.txt

```diff
@@ -151,13 +274,109 @@ PRIVATE
   imgui
 )
 
+set_target_properties(dolphin-emu PROPERTIES PROJECT_LABEL "Dolphin")
+
 if(WIN32)
-  target_sources(dolphin-emu PRIVATE DolphinQt.manifest)
+  if (CMAKE_BUILD_TYPE STREQUAL "Debug")
+    set(Dolphin_NAME "DolphinD")
+  else()
+   set(Dolphin_NAME "Dolphin")
+  endif()
+
+  set_target_properties(dolphin-emu PROPERTIES
+    WIN32_EXECUTABLE TRUE
+    OUTPUT_NAME ${Dolphin_NAME}
+  )
+  target_sources(dolphin-emu PRIVATE DolphinQt.manifest DolphinQt.rc)
+  target_compile_options(dolphin-emu PRIVATE /D _SILENCE_CXX17_RESULT_OF_DEPRECATION_WARNING)
+
+  # Copy files for Windows
+
+  # Copy Sys directory
+  set_property(DIRECTORY APPEND PROPERTY CMAKE_CONFIGURE_DEPENDS "${CMAKE_SOURCE_DIR}/Data/Sys")
+  file(GLOB_RECURSE resources RELATIVE "${CMAKE_SOURCE_DIR}/Data" "${CMAKE_SOURCE_DIR}/Data/Sys/*")
+  foreach(res ${resources})
+    configure_file("${CMAKE_SOURCE_DIR}/Data/${res}" "${CMAKE_RUNTIME_OUTPUT_DIRECTORY}/Release/${res}" COPYONLY)
+    configure_file("${CMAKE_SOURCE_DIR}/Data/${res}" "${CMAKE_RUNTIME_OUTPUT_DIRECTORY}/Debug/${res}" COPYONLY)
+  endforeach()
+
+  # Copy qt.conf.win
+  configure_file(qt.conf.win "${CMAKE_RUNTIME_OUTPUT_DIRECTORY}/Release/qt.conf" COPYONLY)
+  configure_file(qt.conf.win "${CMAKE_RUNTIME_OUTPUT_DIRECTORY}/Debug/qt.conf" COPYONLY)
+
+  file(GLOB_RECURSE Qt5_PLUGIN_STYLES_DLLs "${Qt5_Base}/plugins/styles/*.dll")
+  file(GLOB_RECURSE Qt5_PLUGIN_PLATFORMS_DLLs "${Qt5_Base}/plugins/platforms/*.dll")
+  file(GLOB_RECURSE Qt5_PLUGIN_IMAGEFORMATS_DLLs "${Qt5_Base}/plugins/imageformats/*.dll")
+  file(GLOB_RECURSE Qt5_PLUGIN_STYLES_DEBUG_DLLs "${Qt5_Base}/plugins/styles/*d.dll")
+  file(GLOB_RECURSE Qt5_PLUGIN_PLATFORMS_DEBUG_DLLs "${Qt5_Base}/plugins/platforms/*d.dll")
+  file(GLOB_RECURSE Qt5_PLUGIN_IMAGEFORMATS_DEBUG_DLLs "${Qt5_Base}/plugins/imageformats/*d.dll")
+
+  # Filter out debug DLLs
+  list(FILTER Qt5_PLUGIN_STYLES_DLLs EXCLUDE REGEX ".*d.dll")
+  list(FILTER Qt5_PLUGIN_PLATFORMS_DLLs EXCLUDE REGEX ".*d.dll")
+  list(FILTER Qt5_PLUGIN_IMAGEFORMATS_DLLs EXCLUDE REGEX ".*d.dll")
+
+  # Copy release plugins
```

Comment:
If we must have separate Debug/Release folders, can this be refactored into a function?